torch/_tensor_docs.py

```diff
@@ -3550,6 +3550,13 @@ def callable(a, b) -> number
 In-place version of :meth:`~Tensor.t`
 """)
 
+add_docstr_all('tile',
+               r"""
+tile(*reps) -> Tensor
```

Comment:
I don't think you want the star on reps here?

Reply:
well it works with *reps or a set (because reps is an IntArrayRef, somehow). I've seen other ops are using `*...` too so I just aligned, let me know if you think it's best not to